python-regression/tests/features/steps/response_handling_steps.py

```diff
@@ -1,11 +1,11 @@
-import logging
 from aloe import world, step
 from util.response_logic import response_handling as response_handling
 from util.test_logic import api_test_logic as api_utils
 from util.test_logic import value_fetch_logic
+from util.response_logic import response_handling as response_handling
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [redefinition of unused 'response_handling' from line 2](https://app.codacy.com/manual/GalRogozinski/iri/pullRequest?prid=4339340)